gobblin-temporal/src/main/java/org/apache/gobblin/temporal/yarn/YarnService.java

```diff
@@ -440,7 +430,7 @@ public synchronized boolean requestTargetNumberOfContainers(int numContainers, S
     LOGGER.info("Trying to set numTargetContainers={}, in-use helix instances count is {}, container map size is {}",
         numContainers, inUseInstances.size(), this.containerMap.size());
 
-    requestContainers(numContainers, Resource.newInstance(defaultContainerMemoryMbs, defaultContainerCores));
+    requestContainers(numContainers, Resource.newInstance(defaultContainerMemoryMbs, defaultContainerCores), Optional.absent());
```

Comment:
(WRT the enclosing method...)

does it need to be `public`?  also, if it's only used by `requestInitialContainers` we might name it thus.  on the other hand, why does special case the reading from `config`, rather than doing `WorkerProfile.getConfig`?

Reply:
great catch, was just trying to not modify existing code much so didn't changed, correcting it to use baselineWorker config and making method private as well.

Reply:
removed the method fully.